cwl/src/test/scala/wdl4s/cwl/CwlWorkflowWomSpec.scala

```diff
@@ -124,10 +170,10 @@ class CwlWorkflowWomSpec extends FlatSpec with Matchers {
     val wc = nodes.collectFirst({ case wc: CallNode if wc.name == "wc" => wc }).get
 
     ps.upstream shouldBe empty
-    cgrep.upstream.filter(_ eq ps).size shouldBe 1
-    cgrep.upstream.filter(_ eq cgrepPatternInput).size shouldBe 1
+    cgrep.upstream.count(_ eq ps) shouldBe 1
+    cgrep.upstream.count(_ eq cgrepPatternInput) shouldBe 1
```

Comment:
ToL: Maybe a new matcher than lets us easily do `Set[GraphNode]` reference equality would be useful here?

Reply:
Switched away from reference equality, but is now using a scalatest Matcher.